### PR TITLE
Fixes in Customer and Agent Chat APIs

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
- `check goals` added to table
- `timestamp` replaced with `seen_up_to` + correct reference in tables
- push and webhook `events_marked_as_seen`
- more info in the `get_form` method description 
- file event parameters updated 
- correct response for get chat threads in Agent API

### CODE REVIEW

File: content/messaging/agent-chat-api/index.mdx [hunk 1]
```diff
@@ -826,25 +827,25 @@ Here's the list of all system messages you might come across:
...
+| `created_at`                     | optional  |                                           |
+| `email`                          | optional  |                                           |
 | `fields`                         | optional  | Is not present when the chat is archived. |
-| `name`                           | optional  |                                         - |
-| `last_seen_timestamp`            | optional  |                                         - |
-| `last_visit`                     | optional  |                                         - |
-| `present`                        | optional  |                                         - |
-| `statistics`                     | optional  |                                         - |
+| `name`                           | optional  |                                           |
+| `events_seen_up_to`              |           | RFC 3339 datetime string                  |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx]: Is it also string in RFC format? Required?

File: content/messaging/agent-chat-api/index.mdx [hunk 2]
```diff
@@ -187,7 +187,7 @@ File event informs about a file being uploaded.
 | `custom_id`                                           | optional  |                                                                   - |
 | `properties`                                          | optional  |                                                                   - |
 | `recipients`                                          | required  |                          Possible values: `all` (default), `agents` |
-| `system_message_type`                                 | required  |                                                                   - |
+| `url`                                                 | required  | Has to point to the LiveChat CDN.                                   |
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx]: Does the url in the file event have to point to LC CDN?

Comment (litwinow) [reply content/messaging/agent-chat-api/index.mdx]: Yes. It's also recommended to use url returned by `upload_file`.

File: content/messaging/agent-chat-api/index.mdx (hunk 1 above)
Comment (litwinow) [reply content/messaging/agent-chat-api/index.mdx]: Yes and yes :)

Comment (litwinow) [reply content/messaging/agent-chat-api/index.mdx]: Or actually it's optional (as other fields here)

File: content/messaging/customer-chat-api/index.mdx [hunk 3]
```diff
@@ -2440,28 +2161,14 @@ curl -X POST \
...
-      }'
-``` -->
-
 <Section>
 
 <Text>
 
 ### Get Form
 
+Returns an empty form of a <a href="https://www.livechatinc.com/help/chat-surveys/" target="_blank">prechat or postchat survey</a>.
```
Comment (litwinow) [inline content/messaging/customer-chat-api/index.mdx]: + ticket form

**Pull request merged**